tests/cupyx_tests/scipy_tests/sparse_tests/test_index.py

```diff
@@ -71,7 +71,7 @@ def _run(self, maj, min=None, flip_for_csc=True,
                 actual.data, expected.data)
         else:
             testing.assert_array_equal(
-                actual, expected)
```

Comment:
Why is this change needed?

Reply:
Scipy returns a `numpy.Matrix` object when inner indexing is used. I didn't port that object over to Cupy so we just return an `ndarray`. It took an extra conversion to compare the two directly.

Reply:
`numpy.matrix` also looks deprecated: https://numpy.org/doc/stable/reference/generated/numpy.matrix.html

Reply:
I see. It seems OK to convert to ndarray with `asarray`.
Why do we need to compare after converting to 1d-array?

Reply:
I have not figured out how to get output from inner indexing that are not 1d arrays and I wasn't able to find much for examples in the Scipy tests. I've expanded the dims for the Cupy output to make a more direct comparison. What do you think? Would you rather we do a `numpy.asarray(expected).squeeze()` or call `cupy.expand_dims(val, 0)` before returning the result?